plugins/caddy/configuration.go

```diff
@@ -409,6 +416,8 @@ func parseConfiguration(cfg *Configuration, h *caddyfile.Dispenser, isBlocking b
 			case "log_level":
 				args := h.RemainingArgs()
 				cfg.LogLevel = args[0]
+			case "mode":
+				cfg.DefaultCache.Mode = strings.Join(h.RemainingArgs(), "")
```

Comment:
That's kinda weird. Shouldn't you reject it if there's more than one arg? I think you should do `if len(h.Args()) > 1 { return h.ArgErr() }`